.github/ISSUE_TEMPLATE/config.yml

```diff
@@ -0,0 +1,8 @@
+blank_issues_enabled: false
+contact_links:
+- name: Localisation
+  url: https://github.com/OpenRCT2/Localisation
+  about: For localisation issues, go to the localisation repository.
```

Comment:
Should we say "localisation/translation"? I fear that for people not involved with this process, they will always think about translation first, rather than localisation.

Reply:
Yeah, and perhaps make `Localisation repository` a link.

Reply:
Not sure if markdown is accepted, and it'll open the same link as this entire item, so I don't think it'll be necessary.

Reply:
Looks like markdown is not supported.